### PR TITLE
Archive CVS LTC vaccination data

### PR DESCRIPTION
**Pull request by zachlipton**

### CODE REVIEW

File: .github/workflows/update_hhs.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ jobs:
         curl https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=vaccination_data -o data/cdc_vaccinations.json
         curl https://data.cdc.gov/api/views/9mfq-cb36/rows.csv?accessType=DOWNLOAD -o data/cdc_cases_deaths.csv
         curl https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=integrated_county_latest_external_data -o data/cdc_counties.csv
+        curl https://cvshealth.com/sites/default/files/cvs-health-covid-19-vaccination-data.pdf -o data/cvs_ltc_vaccinations.pdf
```
Comment (julia326) [inline .github/workflows/update_hhs.yml:32]: Since this is a PDF, I wonder if we want to do this with screenshots instead?

Comment (zachlipton) [reply .github/workflows/update_hhs.yml:32]: I was wondering that. Whichever is easiest—I wasn't sure if we had an easy place to stash screenshots for something that isn't a state. I'm good with whatever you'd prefer

Comment (julia326) [reply .github/workflows/update_hhs.yml:32]: ah looks like we're already doing this :) https://github.com/COVID19Tracking/screenshots/blob/main/configs/vax/US.yaml

Comment (zachlipton) [reply .github/workflows/update_hhs.yml:32]: Well look at that! :)

**Pull request closed without merging**